lib/src/base-styles/_nx-btn.scss

```diff
@@ -35,6 +35,10 @@
     outline-color: var(--nx-swatch-blue-30);
   }
 
+  &:focus-visible {
```

Comment:
When the button is focused and then clicked, the browser sets a temporary default blue outline around the button (I assume because we are using `&:focus:not(:active)`). I discussed this with Jason, and removed the outline here since we already have `:focus` and `:active` states defined for all the buttons, and should therefore pass accessibility.